docs/en-US/component/select.md

```diff
@@ -117,45 +117,46 @@ If the binding value of Select is an object, make sure to assign `value-key` as
+| multiple                                         | whether multiple-select is activated                                                                                        | boolean                                    | true / false                                                                                              | false            |
+| disabled                                         | whether Select is disabled                                                                                                  | boolean                                    | true / false                                                                                              | false            |
+| value-key                                        | unique identity key name for value, required when value is an object                                                        | string                                     | —                                                                                                         | value            |
+| size                                             | size of Input                                                                                                               | string                                     | large/default/small                                                                                       | default          |
+| clearable                                        | whether select can be cleared                                                                                               | boolean                                    | true / false                                                                                              | false            |
+| collapse-tags                                    | whether to collapse tags to a text when multiple selecting                                                                  | boolean                                    | true / false                                                                                              | false            |
+| collapse-tags-tooltip                            | whether show all selected tags when mouse hover text of collapse-tags. To use this, `collapse-tags` must be true            | boolean                                    | true / false                                                                                              | false            |
+| max-collapse-tags<VersionTag version="2.2.30" /> | The max tags number to be shown. To use this, `collapse-tags` must be true                                                  | number                                     | —                                                                                                         | 1                |
```

Comment:
@kooriookami let's move this line to the end of this table, and also we should update the `VersionTag` to `^(2.2.ver)` syntax.

Reply:
Done. :D